Formula/emacs-plus@26.rb

```diff
@@ -57,6 +57,7 @@ def install
     imagemagick_lib_path = Formula["imagemagick@6"].opt_lib/"pkgconfig"
     ohai "ImageMagick PKG_CONFIG_PATH: ", imagemagick_lib_path
     ENV.prepend_path "PKG_CONFIG_PATH", imagemagick_lib_path
+    ENV.append "CFLAGS", " -DFD_SETSIZE=10000 -DDARWIN_UNLIMITED_SELECT "
```

Comment:
you don't need these extra spaces around

Reply:
> you don't need these extra spaces around

fixed.